src/views/donationItemView/index.tsx

```diff
@@ -96,20 +95,63 @@ export default function DonationItemView({ donations }: DonationItemProps) {
         )
       : donations;
 
-  const uniqueDonationItems = Array.from(
-    new Set(filteredDonations.map((donation) => donation.items).flat(1))
+  // Grab all donation items from all donations and make a single flat array
+  const flatItems = filteredDonations.flatMap((donation) => donation.items);
+
+  // Then, group items by name and aggregate quantities and values
+  const groupedItems = flatItems.reduce(
+    (
+      acc: Record<
+        string,
+        {
+          item: { name: string; category: string; _id: string };
+          quantity: number;
+          totalValue: number;
+          barcode?: string;
+          evaluation: string;
+          itemIds: string[];
+        }
```

Comment:
Is there a type we can use here instead? This is very prone to breaking and it's unclear what it represents